pkg/agent/openflow/pipeline.go

```diff
@@ -875,33 +875,61 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 
 // conjunctionActionFlow generates the flow to jump to a specific table if policyRuleConjunction ID is matched. Priority of
 // conjunctionActionFlow is created at priorityLow for k8s network policies, and *priority assigned by PriorityAssigner for CNP.
-func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.TableIDType, nextTable binding.TableIDType, priority *uint16) binding.Flow {
+func (c *client) conjunctionActionFlow(conjunctionID uint32, tableID binding.TableIDType, nextTable binding.TableIDType, priority *uint16, enableLogging bool) binding.Flow {
 	var ofPriority uint16
 	if priority == nil {
 		ofPriority = priorityLow
 	} else {
 		ofPriority = *priority
 	}
 	conjReg := IngressReg
-	if tableID == EgressRuleTable {
+	if tableID >= EmergencyEgressRuleTable && tableID <= EgressRuleTable   {
```

Comment:
Please use `GetCNPEgressTables()`. During this release the CNP egress tables will be altered from time to time and this function will always be up to date on which tables are reserved for CNP.